examples/client/Locomotion/src/context/newRideContext/utils.ts

```diff
@@ -165,11 +165,12 @@ export const formatStopPointsForEstimations = (requestStopPoints: any[]) => requ
   lng: sp.lng,
 }));
 
-export const getFormattedPrice = (priceCurrency: string | undefined, priceAmount: number) => {
+export const getFormattedPrice = (priceCurrency: string | undefined,
+  priceAmount: number | undefined) => {
```

Comment:
instead of explicitly add undefined in the type, you can mark it as optional (with ? )

Reply:
There are many options here, I could also set default value as 0, I just wanted to match it with priceCurrency